README.md

```diff
@@ -8,8 +8,22 @@ Node.js application that connects volunteers with volunteers/projects based on t
 
 ## Requirements
 
-- [Node.js](https://nodejs.org) - version 8.4 or higher (if you have nvm type `nvm use` in the repo to use the correct version)
-- graph.cool CLI: `npm install -g graphcool`
+
+- [Node.js](https://nodejs.org) - version 8.4 (if you have nvm type `nvm use` in the repo to use the correct version)
+
+Setup:
+1. clone repo
+2. cd into repo directory
+3. `npm install`
+4. Follow [these instructions to set up Auth0](https://github.com/graphcool/templates/tree/master/auth/auth0#3-setup-auth0) for the project.
```

Comment:
Change "localhost:8080" to "localhost:3000" for new screenshot of auth0 profile setup